PowerEditor/src/Parameters.h

```diff
@@ -1545,7 +1545,7 @@ class NppParameters final
 	TiXmlDocument * getCustomizedToolIcons() const {return _pXmlToolIconsDoc;};
 
 	bool isTransparentAvailable() const {
-		return (_transparentFuncAddr != NULL);
+		return (_winVersion >= WV_VISTA);

```

Comment:
[SetLayeredWindowAttributes](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setlayeredwindowattributes) is supported from Windows 2000 Professional but Notepad++ doesn't check if Windows 2000 is Professional or not.

Reply:
@donho 
btw
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/6a8a446eb61f96b00f74ae5baa0894ff16e7b18f/PowerEditor/src/Parameters.cpp#L7623
does not return "Windows 11"

Reply:
@ozone10 
OK, nice caught. I'll add case WV_WIN11.